core/mobile/src/backend/MobileRpcServer.ts

```diff
@@ -102,6 +103,20 @@ export class MobileRpcServer {
     MobileRpcServer.interop.sendBinary = sender;
   }
 
+  private _sendPending() {
+    if (this._pendingMessages === undefined)
+      return;
+
+    for (const message of this._pendingMessages) {
+      this._connection!.send(message, (err) => {
+        if (err) {
+          throw err;
```

Comment:
Isn't this going to cause problems? I'm not sure what a solution would be, but throwing the error is going to break the construction of this RPC server instead of simply propagating a message failure to wherever it would normally go.

Reply:
I just realized that the error will be thrown from the callback, which won't propagate up to the constructor. (Having said that, where _will_ it go?)

Reply:
The constructor isn't blocked by processing the pending messages. That doesn't happen until the websocket is connected.

Reply:
Sorry, I should have said that _onConnection wouldn't finish, not the constructor. But since the error happens in a callback, I suspect that the throw just goes into the ether. If that is the case, then simply ignoring the error would be better. If that's not the case, where does the throw end up?

Reply:
I don't know where the error goes, I just followed the existing pattern in _createSender.

Reply:
I feel that that pattern is wrong in both locations, but given that as a precedent, I'll withdraw my complaint.